### ISSUE TITLE
Ad-hoc extension does not work

### ISSUE THREAD

**Issue by hlb** [Compass/compass#1200]
The "Ad-hoc extension" feature does not work in both Compass 0.12.2 and 0.13.alpha 2 http://compass-style.org/help/tutorials/extensions/

https://gist.github.com/hlb/5087139

Or is there anything I have missed?

**Comment by scottdavis** [comment]
will get this fixed for 0.13

**Comment by robwierzbowski** [comment]
+1. Ad hoc extensions are a perfect solution for front-end package managers like Bower. Right now we're limited to using Sass components, but when this is fixed we'll have access to Compass extensions as components for app builds.

This works for me in 0.12.2, but I've heard it doesn't in 0.13.alpha.4.

**Comment by cimmanon** [comment]
I just updated Compass to 0.13.alpha.4 (Markab) and now one of my ad hoc extensions is ignored and it appears to be because of the name of the directory.

```
// doesn't work
extensions/cimma
// works
extensions/cimm
extensions/cimma-bak
extensions/cimma-
extensions/cimmaa
```

Error from the console:

```
>>> Compass is watching for changes. Press Ctrl-C to Stop.
    error sass/test.scss (Line 1: File to import not found or unreadable: cimma.
Load paths:
  /home/chris/compass-extensions/sass
  /usr/local/lib/ruby/gems/1.9.1/gems/compass-0.13.alpha.4/frameworks/compass/stylesheets
  /home/chris/compass-extensions/extensions/foo/stylesheets
  /home/chris/compass-extensions/extensions/gmedia/stylesheets
  /home/chris/compass-extensions/extensions/normalize/stylesheets
  Compass::SpriteImporter)
```

My load paths listing doesn't even include `/home/chris/compass-extensions/extensions/cimma/stylesheets`, but all of my other ad hoc extensions are listed.  This worked without issue in previous versions of Compass.

**Comment by cimmanon** [comment]
I found the problem.  I had a new extension where I missed renaming it in lib/_my-extension_.rb.

**Comment by chriseppstein** [comment]
This should be fixed in 1.0.0 please test against the alphas.
